cate/ds/__init__.py

```diff
@@ -63,19 +64,24 @@ def cate_init():
             store_configs = yaml.safe_load(fp)
 
     for store_name, store_config in store_configs.items():
-        if store_config.get('store_id', '') == 'file' \
+        store_id = store_config.get('store_id')
+        assert_given(store_id, name='store_id', exception_type=RuntimeError)
+
+        if store_id == 'file' \
                 and 'store_params' in store_config \
                 and store_config.get('store_params', {}).get('root') is None:
-            base_dir = os.environ.get('CATE_LOCAL_DATA_STORE_PATH',
-                                      os.path.join(get_data_stores_path(),
-                                                   store_name))
-            store_config['store_params']['root'] = base_dir
-        store_params_schema = get_data_store_params_schema(
-            store_config.get('store_id')
-        )
+            root = os.environ.get('CATE_LOCAL_DATA_STORE_PATH',
+                                  os.path.join(get_data_stores_path(),
+                                               store_name))
```

Comment:
Didn't we agree on that we wanted to get rid of this global variable?

Reply:
This was not goal of this PR. If not set, the default is used, which should now work correctly (with current user "xcube").

Reply:
What would be the point of this PR then? To increase the version to 3.1? The change in the changelog has not been considered yet.

Reply:
Use of xcube 0.9

Reply:
> What would be the point of this PR then? To increase the version to 3.1?

Yes. See title of PR.